src/compiler/program.ts

```diff
@@ -1257,10 +1268,14 @@ namespace ts {
             };
         }
 
-        function getProjectReferences() {
+        function getResolvedProjectReferences() {

```

Comment:
IMO this is a good change, but still it's a breaking API change and should be documented somewhere